tasks/affective_task/utils/timer.py

```diff
@@ -28,9 +28,5 @@ def timer(seconds: int, callbacks: List[Callable], pre_text: str, screen, Rating
             seconds_left_to_count = 0 if seconds_left_to_count < 0.0 else int(seconds_left_to_count)
             TIM = seconds_left_to_count
             render_text_center(pre_text + str(seconds_left_to_count + 1), (300, 50), screen, y_offset=-420)
-        
-        if pre_text == "Team: " and Rating == False:
```

Comment:
Remove the parameter `Rating = False` from the function `timer`. It is not used anywhere in the function.